tests/admin_scripts/tests.py

```diff
@@ -106,6 +106,25 @@ def remove_settings(self, filename, is_dir=False):
         if os.path.isdir(cache_name):
             shutil.rmtree(cache_name)
 
+    def write_configured_settings(self, extra='', args=''):
+        template = os.path.join(
+            os.path.dirname(__file__), 'configured_settings_manage.py-tpl')
+        destination = os.path.join(
+            os.path.dirname(__file__), 'configured_settings_manage.py')
+        shutil.copyfile(template, destination)
+
+        with open(destination, 'r') as fp:
+            contents = fp.read()
+        contents = contents.replace('{{ extra }}', extra)
+        contents = contents.replace('{{ args }}', args)
+        with open(destination, 'w') as fp:
+            fp.write(contents)
+
+    def remove_configured_settings(self):
+        path = os.path.join(
```

Comment:
Use a single line. Our [style guide](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style) allows lines up to 119 characters if it helps readability.